src/page/main/main.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import MySearch from './../../component/UI/input/MySearch';
+import catsData from './../../assets/data/cats';
+import Card from './../../component/UI/card/Card';
+import './../main/main.module.css';
+import SearchProps from './../../component/UI/input/interface';
+
+class Main extends React.Component {
+  prop: SearchProps = { type: 'search', placeholder: 'Input some body...' };
+  render() {
+    return (
+      <>
+        <h2>Paws</h2>
+        <MySearch {...this.prop} />
+        <Cards />
+      </>
+    );
+  }
+}
+
+function Cards() {
```

Comment:
```suggestion
const CardList = (({ catsData })) => <div>...</div>
```
It is possible to use shorter construction here

Reply:
yes, well